db/src/tx.rs

```diff
@@ -115,6 +117,8 @@ use types::{
 };
 use upsert_resolution::Generation;
 
+pub type AttributeSet = BTreeSet<Entid>;
```

Comment:
I'm really considering writing `VecSet`, 'cos I think for a lot of our really small sets it'll be cheaper to do linear search.